framework/src/main/java/org/apache/drill/test/framework/CmdParam.java

```diff
@@ -52,10 +52,13 @@
     @Parameter(names = {"-r", "--report"}, description = "Generate json report", required=false)
     public boolean generateReports = false;
 
+    @Parameter(names = {"-p", "--percentThresholdForRandomFailure"}, description = "if less than p% of tests fail, they will be rerun to isolate random failures,max = 0.1", required=false)
```

Comment:
Change to "If less than p% (0.01 - 0.10) of tests fail, they will be rerun to isolate random failures"